apps/condo/domains/billing/components/Settings/IntegrationPanel.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colors, shadows } from '@condo/domains/common/constants/style'
+import { Tooltip, Typography, Tag } from 'antd'
+import { useIntl } from '@core/next/intl'
+import { useRouter } from 'next/router'
+
+export type CardStatuses = 'available' | 'inProgress' | 'chosen' | 'disabled'
+
+interface IIntegrationPanelProps {
+    integrationId: string
+    title: string
+    shortDescription: string
+    status: CardStatuses
+}
+
+const CardContainer = styled.div`
+  position: relative;
+  height: 229px;
+  box-sizing: border-box;
+  border: 1px solid ${colors.lightGrey[5]};
+  border-radius: 8px;
+  padding: 24px;
+  transition: all 0.2s ease-in-out;
+  cursor: pointer;
+  background-color: ${props => (props['data-status'] !== 'disabled' && props['data-status'] !== 'available') 
+        ? colors.lightGrey[4] 
+        : 'transparent'};
+  &:hover {
+    border-color: transparent;
+    ${shadows.elevatedShadow}
+  }
+`
+
+const TwoLineClamp = styled.div`
+  display: -webkit-box;
+  overflow: hidden;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  text-overflow: ellipsis;
```

Comment:
Try to find this component in ant'd design typography components

Reply:
Changed to Typography.Paragraph